### DISCUSSION TITLE
Count_all_tweets function: How to search for two keyword conditions

### DISCUSSION THREAD

**Discussion by qehuang** [cjbarrie/academictwitteR#241] [Q&A] [answered]
### Describe the solution you'd like

Hello,

I wonder if it is possible to use count_all_tweets to count the tweets that contain at least one word from wordlist1 and at least one word from wordlist2. Please advise, thank you!

**Comment by chainsawriot** [comment, marked as answer]
@qehuang Read [this](https://developer.twitter.com/en/docs/twitter-api/tweets/search/integrate/build-a-query) first.

If you still can't find the answer: (A OR B) (C OR D)

**Reply by qehuang** [reply]
Thanks. I tried that earlier but now I realize it should be "(A OR B) (C OR D)"

**Reply by chainsawriot** [reply]
@qehuang Could you mark this as answered?
